go.mod

```diff
@@ -156,3 +156,5 @@ require (
 // the android OS using gomobile. This replace can be removed once any
 // of those projects update their github.com/lib/pq dependency.
 replace github.com/lib/pq => github.com/lib/pq v1.10.4
+
+replace github.com/crypto-power/instantswap => github.com/vibros68/instantswap v0.0.0-20231205093815-045a3f9f0a0c
```

Comment:
No, submit a PR to instantswap.

Reply:
It is for reviewing. After the review is done, we will merge on both instantswap and cryptopower.
@dreacot please confirm on it

Reply:
Okay, I’ve seen your instantswap PR.